templates/sequelize/sequelize.js

```diff
@@ -1,4 +1,4 @@
-const config = require('./dbConfig');
+const config = require('../../config/default.json');
```

Comment:
You should not import config files when using node config. 

Check out this quick start for an example https://github.com/lorenwest/node-config#quick-start